backend/app/api/ws/chat_namespace.py

```
@@ -78,6 +78,80 @@
 
+def _query_active_team_by_id(db: Session, team_id: Optional[int]) -> Optional[Kind]:
+    """Load an active Team by database id."""
+    if not team_id:
+        return None
+
+    return (
+        db.query(Kind)
+        .filter(
+            Kind.id == team_id,
+            Kind.kind == "Team",
+            Kind.is_active == True,
```

Ruff (0.15.15)

[error] Avoid equality comparisons to `True`; use `Kind.is_active:` for truth checks

Replace with `Kind.is_active`

(E712)